### ISSUE TITLE
Can't login to my account when opening Raid using this app.

### ISSUE THREAD

**Issue by cmarabate** [FarbstoffRSL/RSL-Helper#18]
If I let this app open Raid for me, it doesn't seem to be able to login to my account. I just get this error over and over...
![image](https://user-images.githubusercontent.com/12236062/118136625-507ff800-b3d2-11eb-9848-8a815c2fd5fe.png)

**Comment by Dwza** [comment]
This looks more like inet issue or plarium doing some Updates. May try in about an hour and respond again.
:)

**Comment by BlackHawk3000** [comment]
This is not a problem of the clicker.

**Comment by cmarabate** [comment]
I am able to login via BlueStacks & Plarium Play... I only have this issue when I open this app and let it open Raid for me.

**Comment by Dwza** [comment]
what is your windows version?

**Comment by cmarabate** [comment]
@Dwza Windows 10...
![image](https://user-images.githubusercontent.com/12236062/118326833-835de500-b4d3-11eb-90ed-8a203df995b7.png)

I am able to use the Autoklicker if I already have Plarium Play open and then I open this app afterward but I notice a ton of issues when opening it this way...

1) When I click on one of my champs that get pulled into the Autoklicker, I can't see any of their Equipment like ChoseN was able to in his video...
![image](https://user-images.githubusercontent.com/12236062/118325607-b4d5b100-b4d1-11eb-955a-1819dfe4a446.png)

2) Not sure what the Event Tracker is supposed to look like but it is blank for me...
![image](https://user-images.githubusercontent.com/12236062/118325691-ce76f880-b4d1-11eb-8686-76786ca929bd.png)

3) Shard Tracker not working...
![image](https://user-images.githubusercontent.com/12236062/118325730-e3538c00-b4d1-11eb-918d-a1e69f38f51e.png)

4) I set this once but it did not seem to do anything and it apparently was not saved because it is gone again...
![image](https://user-images.githubusercontent.com/12236062/118325813-03834b00-b4d2-11eb-932b-2262f92a9e75.png)

5) The Tools page does not seem to do anything...
![image](https://user-images.githubusercontent.com/12236062/118325986-480ee680-b4d2-11eb-9b58-6656fc77ff5f.png)

6) I don't know how the CB Simulator is supposed to work but I don't think it is working either...
![image](https://user-images.githubusercontent.com/12236062/118326543-1d715d80-b4d3-11eb-8250-b779e67766aa.png)

I'm sure this is all because I already have Raid open and not letting this app open it for me but like I said, I am unable to login when I let the app launch Raid for me.

Thanks!

**Comment by Dwza** [comment]
ChoseN uses this tool?

**Comment by BlackHawk3000** [comment]
> ChoseN uses this tool?

Ja tut er, daher der "Ansturm". 

Did you start the Helper before RSL and make a run or two?

**Comment by cmarabate** [comment]
> ChoseN uses this tool?

Yeah, he discovered it recently and did a video about it...
[https://www.youtube.com/watch?v=mR5_vq4fjNc&t=484s](https://www.youtube.com/watch?v=mR5_vq4fjNc&t=484s)

**Comment by cmarabate** [comment]
> > ChoseN uses this tool?
> 
> Ja tut er, daher der "Ansturm".
> 
> Did you start the Helper before RSL and make a run or two?

No. Like I said, I can't login to my account if I start the helper and let it open Raid for me. I have to already have Plarium Play open before starting the helper to use the autoklicker but nothing else works.

**Comment by Dwza** [comment]
have you granted admin prev's for the helper?

**Comment by cmarabate** [comment]
> have you granted admin prev's for the helper?

I am opening it as Administrator. Is that what  you mean?

**Comment by Jester-D** [comment]
I have exactly the same problem of not being able to login if I specify the path for Raid.exe.  

What I did was, remove the location path from within the RSLHelper.ini file (so it showed "PathRoTaid=" only.  Save the file and then launch the RaidHelper as an Administrator.

When it loads and says it couldn't find the raid.exe, I manually launched Raid via the Plarium Play app.  

The RaidHelper identified the window had been launched and, after a run though campain, it populated all of the data.
Shards and mercy plus Events are showing up as normal.

Hope that helps people out.

**Comment by Dwza** [comment]
The Path to Raid is may a lil bit confusing.... actually you should point it to your plariumplay.exe. 
e.g. **C:\Gaming\Plarium\PlariumPlay\PlariumPlay.exe** Than the rsl-helper will find the raid.exe on it's own. I will talk to farbstoff to make this more clear by adding some kinde of hint text or so. When adding the right path, does this solves the problem?

**Comment by Jester-D** [comment]
Ah, interesting!  When I point it to plariumplay.exe, I can login and everything works successfully.  

So definitely it may be an idea to reword "Install Directory of Raid" to "Install Directory of Plarium Play" in the Setting part of the App

**Comment by cmarabate** [comment]
Yup, that was the issue!

I naturally assumed the Install Directory of Raid meant the directory Raid.exe was in. After opening the app setting that to the path that PlariumPlay.exe is in, then restarting the app; everything worked fine.

When I looked at the settings page again it looked like the Install path that I set was not saved because nothing shows up in the input but apparently it is saved because I restarted the app again and was able to login.
![image](https://user-images.githubusercontent.com/12236062/118478048-6b59b180-b6dd-11eb-9955-5ac1e45ab669.png)

Thanks!
